### ISSUE TITLE
new "Transpose" ops when transform keras to onnx

### ISSUE THREAD

**Issue by yanghanxy** [onnx/onnxmltools#130]
Hi, 

Thank you for sharing this nice tool for converting keras to onnx. 

I followed the [examples](https://github.com/onnx/onnxmltools#examples) to convert my keras model to onnx model, but encountered with some weird results. 

The overall model structure keeps the same before and after the conversion, but there's a lot of new "Transpose" ops in the converted onnx model. 
Especially, there will be a new "Transpose" op before and after [BatchNorm, Padding, Conv] ops.

Do you know what may cause this weird result? How can I eliminate those "Transpose" ops?

Here is the plotted keras and ONNX model:
[keras](https://user-images.githubusercontent.com/5886506/44519642-340e6f80-a700-11e8-8243-8cc387132bad.png)
[onnx](https://user-images.githubusercontent.com/5886506/44519643-340e6f80-a700-11e8-96ee-bd39ae502011.png)

**Comment by wschin** [comment]
@yanghanxy , thank you for reporting this problem. Could you please change the `axis` attribute of Keras BatchNorm to `1`? [Here](https://github.com/onnx/onnxmltools/blob/0cd78a742837aa429c57d3a54f269a2421770cca/onnxmltools/convert/keras/operator_converters/BatchNorm.py#L16) and [here](https://github.com/onnx/onnxmltools/blob/0cd78a742837aa429c57d3a54f269a2421770cca/onnxmltools/convert/keras/operator_converters/BatchNorm.py#L55) we use the `axis` attribute to determine if a Transpose is needed.

**Comment by yanghanxy** [comment]
Hi @wschin, 

Thank you for your reply! I'll set the BachNorm in keras according to onnx's format requirement. 
Seems that my problem got sloved, closing this issue.
